### DISCUSSION TITLE
Kindly support HINDI language

### DISCUSSION THREAD

**Discussion by DhiruBhai5** [xxyzz/WordDumb#264] [Ideas]
This plugin is game changer and truly great. Kindly add Hindi language support, for Hindi books, will contribute if hindi language support is added.

**Comment by xxyzz** [comment]
Add English and other Word Wise gloss languages for Hindi books could be done but spaCy doesn't have trained Hindi NER model so X-Ray can't be supported.

**Reply by DhiruBhai5** [reply]
Word wise for hindi is enough for now

**Reply by xxyzz** [reply]
Please download test version at here: https://github.com/xxyzz/WordDumb/actions/runs/10997056664
